docs/getting-started/using-ale/ale-quickstart.md

```diff
@@ -0,0 +1,54 @@
+# ALE Quick Start
+
+This document provides a set of steps to get setup for generating Image Support
+Data (ISD) for an image.
+
+## Installation
+
+Install ALE with conda. (If you don't have conda, [get it via miniforge](https://conda-forge.org/download/).)
+```sh
+    conda create -n ale
+    conda activate ale  # Activate your environment every time you need to use ALE.
+    conda install -c conda-forge ale
+```
```

Comment:
it might not completely be overlapping with other docs, but I feel the "generate ISD" docs would be more useful? Way we should prefer that.